lib/spacesuit/api_message.ex

```diff
@@ -5,7 +5,7 @@
 # Just does JSON for now
 defmodule Spacesuit.ApiMessage do
   @derive [Poison.Encoder]
-  defstruct [:status, :message]
+  defstruct [:errorCode, :errorMessage]
```

Comment:
the model platform uses, `errorCode` is frequently used, the other one could be changed / dropped